### PR TITLE
Refresh when  ENV of kubeconfi changes, not just .kube

### LINKED ISSUE #41: Support refresh when kubeconfig ENV changes
There is code written to support .kube changes but nothing for using ENV - KUBECONFIG.

### PR DESCRIPTION
**Pull request by joshuawilson**
fixes #41

### CODE REVIEW

File: src/tree/serviceExplorer.ts [hunk 1]
```diff
@@ -11,6 +11,12 @@ import { ServiceDataProvider } from './serviceDataProvider';
 import { WatchUtil, FileContentChangeNotifier } from '../util/watch';
 
 const kubeConfigFolder: string = path.join(Platform.getUserHomePath(), '.kube');
+const kubeconfigSplit: string[] = process.env.KUBECONFIG.split('/');
```
Comment (gorkem) [inline src/tree/serviceExplorer.ts:14]: I think this needs to use `path.delimiter` as `process.env.KUBECONFIG.split(path.delimiter);`

File: src/tree/serviceExplorer.ts [hunk 2]
```diff
@@ -28,10 +34,14 @@ export class ServiceExplorer implements Disposable {
 
   private fsw: FileContentChangeNotifier;
 
+  private fswEnv: FileContentChangeNotifier;
+
   public constructor() {
     const treeDataProvider = new ServiceDataProvider();
     this.fsw = WatchUtil.watchFileForContextChange(kubeConfigFolder, 'config');
+    this.fswEnv = WatchUtil.watchFileForContextChange(kubeConfigFolderENV, 'kubeconfig');
```
Comment (gorkem) [inline src/tree/serviceExplorer.ts:42]: KUBECONFIG points to one or more files [1] . I do not think you should assume a name here.

https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/#the-kubeconfig-environment-variable

Comment (joshuawilson) [reply src/tree/serviceExplorer.ts:42]: I didn't realize. Thanks for point that out.

File: src/tree/serviceExplorer.ts (hunk 1 above)
Comment (joshuawilson) [reply src/tree/serviceExplorer.ts:14]: good catch

**Pull request merged**